integration-tests/modules/bank_test.go

```diff
@@ -24,6 +24,257 @@ import (
 
 var maxMemo = strings.Repeat("-", 256) // cosmos sdk is configured to accept maximum memo of 256 characters by default
 
+// TestBankBurn tests burning native tokens using AssetFT MsgBurn.
+func TestBankBurn(t *testing.T) {
```

Comment:
looks to me that this test overlaps or is a duplicate of the tests in assetft_test.go. merge the 2 tests.